src/structure/viewer.ts

```diff
@@ -891,23 +879,29 @@ export class MoleculeViewer {
             const property: string = this._options.color.property.value;
             const mode: string = this._options.color.mode.value;
 
-            if (mode === 'log') {
-                properties[property] = properties[property].map((value) =>
-                    !isNaN(Number(value)) ? Math.log10(Number(value)) : NaN
-                );
-            } else if (mode === 'sqrt') {
-                properties[property] = properties[property].map((value) =>
-                    !isNaN(Number(value)) ? Math.sqrt(Number(value)) : NaN
-                );
-            } else if (mode === 'inverse') {
-                properties[property] = properties[property].map((value) =>
-                    !isNaN(Number(value)) ? 1 / Number(value) : NaN
-                );
+            if (property !== 'element' && mode === ('log' || 'sqrt' || 'inverse')) {
+                properties[property] = properties[property].map((value) => {
+                    if (value !== null) {
```

Comment:
where would a `null` value comes from?

Reply:
The null value came when there were no property value assigned on the atom. As for some atoms and properties in the Zeolite example.